UnitTests/ComplexResult.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
+
+public class ComplexResult : IEquatable<ComplexResult>
```

Comment:
Ha! I kept wondering why this class is here and then I realized, this is for the unit tests! Silly me.